packages/manager/modules/pci/src/projects/project/storages/databases/components/time-range/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+import '@ovh-ux/ui-kit';
+
+import component from './time-range.component';
+
+const moduleName = 'ovhManagerPciStoragesDatabasesTimeRange';
```

Comment:
Then, shouldn't it be done in the ui-kit ? If you don't mind, given the short amount of time, would it be ok if we do something about it in the next milestone ?

Reply:
All the components in UI-KIT are public, reused across Manager, web, order, and designed, approved by OVHcloud UX/design team. "manager-components" module is used to store all components reused in Manager. I don't about the complexity but it would be ideal if you can take care of this in this PR.

Reply:
PU has moved some of these components to manager-components module through another PR